scripts/qtcreator/templates/wizards/openframeworks/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include "ofMain.h"
```

Comment:
@arturoc.  This is brilliant. Are you planning to extend this to Qt on OSX?  It would be great to be able to use oF in both Windows and Mac Qt.